lib/main.dart

```diff
@@ -235,6 +246,9 @@ class App extends StatelessWidget {
             '/settings/audio-sources': (context) => const AudioSourcesScreen(),
             '/settings/quick-links': (context) => const QuickLinksScreen(),
             '/settings/backup': (context) => const BackupScreen(),
+            '/settings/events-config': (context) => const EventConfigScreen(),
+            '/settings/follow-config': (context) =>
```

Comment:
Since this is a child of `events-config` we should probably do `/settings/events/follow`